transposer/plugins/org.polarsys.kitalpha.transposer.rules.handler.generation.chain/model/mapping.generationchain

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="ASCII"?>
-<generationChain:GenerationChain xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:genchainExtension="http://www.polarsys.org/kitalpha/emde/genchain/1.0.0" xmlns:generationChain="http://www.eclipse.org/egf/1.0.0/generationChain" name="mapping" factoryComponentName="org.polarsys.kitalpha.transposer.rules.handler.fcore">
+<generationChain:GenerationChain xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:generationChain="http://www.eclipse.org/egf/1.0.0/generationChain" name="mapping" factoryComponentName="org.polarsys.kitalpha.transposer.rules.handler.fcore">
   <elements xsi:type="generationChain:GenerationChain" name="mapping">
-    <elements xsi:type="genchainExtension:EmdeGeneration" name="Emde Generation mapping" modelPath="/org.polarsys.kitalpha.transposer.rules.handler.ecore/model/mapping.ecore" generateModel="true" generateEdit="true" generateEditor="true" pluginName="org.polarsys.kitalpha.transposer.rules.handler.mapping" basePackage="org.polarsys.kitalpha.transposer.rules.handler.ecore"/>
+    <elements xsi:type="generationChain:EmfGeneration" name="EMF Generation mapping" modelPath="/org.polarsys.kitalpha.transposer.rules.handler.ecore/model/mapping.ecore" generateModel="true" generateEdit="true" generateEditor="true" pluginName="org.polarsys.kitalpha.transposer.rules.handler.mapping" basePackage="org.polarsys.kitalpha.transposer.rules.handler.ecore"/>
```

Comment:
Can you please document this change in the PR/commit text ?

Reply:
Why EmdeGeneration is replaced by EmfGeneration ?

Reply:
Because Transposer does not depends on EMDE!